mmv1/templates/terraform/pre_delete/oracledatabase_autonomous_database.go.tmpl

```diff
@@ -0,0 +1,3 @@
+if d.Get("deletion_protection").(bool) {
+	return fmt.Errorf("cannot destroy instance without setting deletion_protection=false and running `terraform apply`")
```

Comment:
drive-by: we probably want to apply a similar change to https://github.com/GoogleCloudPlatform/magic-modules/pull/12296#discussion_r1847173127 here

Reply:
Yep, please apply this fix!